concepts/05 UI Components/TreeList/00 Getting Started with TreeList/10 Customize Columns/0 Customize Columns.md

```diff
@@ -0,0 +1 @@
+To customize columns, declare the [columns](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/) array. This array can contain objects (column configurations) and text strings (data field names). You can use text strings if you do not need to specify column properties except the [dataField](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#dataField).
```

Comment:
`To customize columns, declare the [columns](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/) array.`

I'd like to switch the sentence order since it's the first sentence in the paragraph. Can we change it to the following?

`Declare the [columns](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/) array to customize columns.`